packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3089,9 +3089,10 @@ class InputDecoration {
   /// This can be used to modify the [BoxConstraints] surrounding [prefixIcon].
   ///
   /// This property is particularly useful for getting the decoration's height
-  /// less than 48px. This can be achieved by setting [isDense] to true and
-  /// setting the constraints' minimum height and width to a value lower than
-  /// 48px.
+  /// less than the minimum tappable height (which is 48px when the visual
```

Comment:
Super nit: Wherever you say "minimum tappable height", I think it should be changed to "minimum tappable size". I think it makes it more logically consistent when you refer to "height and width" later in this paragraph:

```diff
- setting the constraints' minimum height and width to a value lower than the minimum tappable height.
+ setting the constraints' minimum height and width to a value lower than the minimum tappable size.
```

I think that sounds better so you're not comparing "height and width" to "height".